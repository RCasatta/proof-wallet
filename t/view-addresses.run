#!/bin/bash
set -e

$PROOFWALLET view-addresses -m 2 -n 3 --regtest << INPUT 2>&1
y
y
y
y
y
y
viable finger bless corn tomorrow emotion kangaroo there youth impose innocent december swamp reduce famous enter worth search home journey volcano clerk consider noise
[3b6b014e]tpubD6NzVbkrYhZ4Wnsni4GBz5MepuDt6zidxNKZUGKQGgokfadLG2Jf33PXRGSfTCC3xjvv2YhCpFGTrShdUq8mCqZPUjpAu9bnPuAf4vt9Apw
[41f408e8/1']tpubD8kF7X4sjY6y5zRgmYhjd4878gYVtuesSxo8zhYkNdb5KRZK7tU42yS2RTBA6Mn86VRfRXk6RqJ6FQ35tDaLxyhEGG2SSKAf1ifcYG9fNjX
[2eb4fc90/48'/1'/0'/2']tpubDFFFhev78PB1E9LUcWoeBrwJUQrhH23f2ShmW4DueA6WDkd27qgyJeWNL62FcNQHbh6wAJhGhYm2g3xQidQ6G2SpNBh8qon3HLhc6vYzyJy
0
1
2
3
4
5
6
7
8
9
not supported
NEXT
NEXT
NEXT
PREV
PREV
PREV
PREV
CHANGE
QUIT
INPUT
