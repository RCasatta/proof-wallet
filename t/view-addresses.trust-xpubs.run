#!/bin/bash
set -e

$PROOFWALLET view-addresses -m 2 -n 3 --regtest --trust-xpubs << INPUT 2>&1
y
y
y
y
y
y
[3b6b014e]tpubD6NzVbkrYhZ4Wnsni4GBz5MepuDt6zidxNKZUGKQGgokfadLG2Jf33PXRGSfTCC3xjvv2YhCpFGTrShdUq8mCqZPUjpAu9bnPuAf4vt9Apw
[41f408e8/1']tpubD8kF7X4sjY6y5zRgmYhjd4878gYVtuesSxo8zhYkNdb5KRZK7tU42yS2RTBA6Mn86VRfRXk6RqJ6FQ35tDaLxyhEGG2SSKAf1ifcYG9fNjX
[2eb4fc90/48'/1'/0'/2']tpubDFFFhev78PB1E9LUcWoeBrwJUQrhH23f2ShmW4DueA6WDkd27qgyJeWNL62FcNQHbh6wAJhGhYm2g3xQidQ6G2SpNBh8qon3HLhc6vYzyJy
QUIT
INPUT
