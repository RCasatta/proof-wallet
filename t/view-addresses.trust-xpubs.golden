Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
Input 3 valid descriptor keys

Enter descriptor key #1: 
Enter descriptor key #2: 
Enter descriptor key #3: ================================================================================
Derivation Path, Address
../0/0, bcrt1qulldgcyt7yuk9c09t7935gk2meq3ssp7txr2ntyer8gfm4cfy40s7r5ku2 (Enter 0 to save as a QR code in address.png)
../0/1, bcrt1qwghj3vqyldul0c44uy7sy0u5mp8m4qzlsemtwrtw4xmvgam2580ss8y8gx (Enter 1 to save as a QR code in address.png)
../0/2, bcrt1q7md5ks3rkmuqgj49tqnllhpfzter3mj5kafdrtxq4n0cw25q3nwqe4pnqt (Enter 2 to save as a QR code in address.png)
../0/3, bcrt1qjskkwkl4c0xttgew26mucgrwz78g6jqvyyvl673944v447vnk9pqw5sjqt (Enter 3 to save as a QR code in address.png)
../0/4, bcrt1qya20t2g0xxkzpg8e8m8m64u2gds807vcj0mlpu38ds7juh8u9c9sx7drsg (Enter 4 to save as a QR code in address.png)
../0/5, bcrt1qvewfys0rweqkgx98p0cmjgldmfhw5smde7skqjtp2fp0u2yl9xnqrlv3tr (Enter 5 to save as a QR code in address.png)
../0/6, bcrt1qtn73scdqv7h9c8lf2au0h5537e4fzyux0qcrez5tkh98ws5fhp2slq9jte (Enter 6 to save as a QR code in address.png)
../0/7, bcrt1q8r4dn3yk5qlyxffk8e58fcdxu5vkcey357qsvlmqpztreeq30lmscvsgnq (Enter 7 to save as a QR code in address.png)
../0/8, bcrt1qkucptzg2nnnhfc38zk28u0s82u0gd55jjz33ncz52se7nsdt34eqqruxwy (Enter 8 to save as a QR code in address.png)
../0/9, bcrt1qggwdv49ewdrjvt0ltc4kyhg30ssz93g2t97gm55lysnmhd3yhshszp9akm (Enter 9 to save as a QR code in address.png)

Controls:
    'NEXT' -- view next 10 addresses
    'PREV' -- view previous 10 addresses
    'CHANGE' -- toggle to/from change addresses
    'QUIT' -- quit proof wallet

Enter your desired command: 